### PR TITLE
add apache 2 source code headers to css files

### LINKED ISSUE #145: Need apache-2 compatible source code headers in all source files
All source files, other than machine-generated files, need to have the Apache-2 source code header, which you can find here: https://github.com/adobe/starter-repo/blob/master/LICENSE_HEADER.md#javascript-java-c-c-swift-go-objective-c

### PR DESCRIPTION
**Pull request by guillermoarellano**
## Description
Updated the css files with the Apache 2 open source header to meet legal's requirements for Adobe project. 

## Related Issue
Closes #145 

## Motivation and Context
Recommendation from Legal.

## How Has This Been Tested?
Didn't know how to run the test scripts.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by guillermoarellano** [comment]
### CSS files updated

- [x] src\component-skins.css
- [x] src\components.css
- [x] src\spectrum-core.css
- [x] src\spectrum-dark.css
- [x] src\spectrum-darkest.css
- [x] src\spectrum-light.css
- [x] src\spectrum-lightest.css
- [x] src\colorStops\spectrum-dark.css
- [x] src\colorStops\spectrum-darkest.css
- [x] src\colorStops\spectrum-light.css
- [x] src\colorStops\spectrum-lightest.css
- [x] src\commons\index.css
- [x] src\commons\skin.css
- [x] src\commons\layout.css
- [x] src\commons\fonts.css
- [x] src\commons\overlay.css
- [x] src\accordion\index.css
- [x] src\accordion\skin.css
- [x] src\actionmenu\index.css
- [x] src\alert\index.css
- [x] src\alert\skin.css
- [x] src\asset\index.css
- [x] src\asset\skin.css
- [x] src\assetlist\index.css
- [x] src\assetlist\skin.css
- [x] src\avatar\index.css
- [x] src\avatar\skin.css
- [x] src\banner\index.css
- [x] src\banner\skin.css
- [x] src\barloader\index.css
- [x] src\barloader\skin.css
- [x] src\breadcrumb\index.css
- [x] src\breadcrumb\skin.css
- [x] src\button\index.css
- [x] src\button\skin.css
- [x] src\buttongroup\index.css
- [x] src\calendar\index.css
- [x] src\calendar\skin.css
- [x] src\card\index.css
- [x] src\card\skin.css
- [x] src\checkbox\index.css
- [x] src\checkbox\skin.css
- [x] src\circleloader\animation.css
- [x] src\circleloader\index.css
- [x] src\circleloader\skin.css
- [x] src\coachmark\animation.css
- [x] src\coachmark\index.css
- [x] src\coachmark\skin.css
- [x] src\cyclebutton\index.css
- [x] src\cyclebutton\skin.css
- [x] src\decoratedtextfield\index.css
- [x] src\dialog\index.css
- [x] src\dialog\skin.css
- [x] src\dropdown\index.css
- [x] src\dropdown\skin.css
- [x] src\dropindicator\index.css
- [x] src\dropindicator\skin.css
- [x] src\dropzone\index.css
- [x] src\dropzone\skin.css
- [x] src\fieldgroup\index.css
- [x] src\fieldlabel\index.css
- [x] src\fieldlabel\skin.css
- [x] src\icon\index.css
- [x] src\icon\ui-icons.css
- [x] src\illustratedmessage\index.css
- [x] src\illustratedmessage\skin.css
- [x] src\inputgroup\index.css
- [x] src\inputgroup\skin.css
- [x] src\label\index.css
- [x] src\label\skin.css
- [x] src\link\index.css
- [x] src\link\skin.css
- [x] src\menu\index.css
- [x] src\menu\skin.css
- [x] src\miller\index.css
- [x] src\page\index.css
- [x] src\page\skin.css
- [x] src\pagination\index.css
- [x] src\pagination\skin.css
- [x] src\popover\index.css
- [x] src\popover\skin.css
- [x] src\quickaction\index.css
- [x] src\quickaction\skin.css
- [x] src\radio\index.css
- [x] src\radio\skin.css
- [x] src\rating\index.css
- [x] src\rating\skin.css
- [x] src\rule\index.css
- [x] src\rule\skin.css
- [x] src\search\index.css
- [x] src\search\skin.css
- [x] src\searchwithin\index.css
- [x] src\sidenav\index.css
- [x] src\sidenav\skin.css
- [x] src\slider\index.css
- [x] src\slider\skin.css
- [x] src\splitbutton\index.css
- [x] src\splitview\index.css
- [x] src\splitview\skin.css
- [x] src\statuslight\index.css
- [x] src\statuslight\skin.css
- [x] src\steplist\index.css
- [x] src\steplist\skin.css
- [x] src\stepper\index.css
- [x] src\stepper\skin.css
- [x] src\table\index.css
- [x] src\table\skin.css
- [x] src\tabs\index.css
- [x] src\tabs\skin.css
- [x] src\tags\index.css
- [x] src\tags\skin.css
- [x] src\textfield\index.css
- [x] src\textfield\skin.css
- [x] src\toast\index.css
- [x] src\toast\skin.css
- [x] src\toggle\index.css
- [x] src\toggle\skin.css
- [x] src\tooltip\index.css
- [x] src\tooltip\skin.css
- [x] src\treeview\index.css
- [x] src\treeview\skin.css
- [x] src\typography\font.css
- [x] src\typography\index.css
- [x] src\typography\skin.css
- [x] src\underlay\index.css
- [x] src\underlay\skin.css
- [x] src\well\index.css
- [x] src\well\skin.css
- [x] vars\spectrum-dark.css
- [x] vars\spectrum-darkest.css
- [x] vars\spectrum-global.css
- [x] vars\spectrum-large.css
- [x] vars\spectrum-light.css
- [x] vars\spectrum-lightest.css
- [x] vars\spectrum-medium.css
- [x] vars\spectrum-middark.css
- [x] vars\spectrum-midlight.css

File: vars/spectrum-dark.css [hunk 1]
```diff
@@ -1,3 +1,15 @@
+/*
```
Comment (lazd) [inline vars/spectrum-dark.css]: Everything in `vars/` is technically machine-generated (though that's not obvious here), and they'll be overwritten each time we update these vars...

**Comment by lazd** [comment]
Will go ahead and merge this and let the headers in `vars/*` get clobbered when we update. Thanks @guillermoarellano !

**Comment by guillermoarellano** [comment]
Thank you Larry.

**Pull request merged**